server/templates/static/about.html

```diff
@@ -24,39 +24,77 @@
+  done over and over.     Our goal with Data Commons is to address this problem.
     </p>
 
     <p>
-      Data Commons attempts to synthesize a single graph from
+      Data Commons synthesizes a single  graph from
       these different data sources. It links references to the same entities (such
       as cities, counties, organizations, etc.) across different datasets to nodes
       on the graph, so that users can access data about a particular entity
-      aggregated from different sources without data cleaning or joining. Like the
-      Web, Data Commons is open - any user can contribute datasets or build
-      applications powered by the graph. In the long term, we hope the data
+      aggregated from different sources without data cleaning or joining. We hope the data
       contained within Data Commons will be useful to students, researchers, and
-      enthusiasts across different disciplines. Though we've "jump-started" the
-      graph with data from publicly available sources (CDC, US Census, FBI, etc.),
-      we encourage you to join and contribute.
+      enthusiasts across different disciplines. 
     </p>
+<br><br>
+    <h2>Who can use it?</h2>
+    <p>
+    Data Commons can be accessed by anyone via  the tools available on
+    datacommons.org. Students, researchers and developers can use the
+    REST and Python APIs, both of which are free for educational,
+    academic and journalistic research purposes. At this point, use of
+    these APIs does not require an API Key.
+</p>
+<p>
+    The data may also be accessed via the SPARQL query interface,
+    which requires an API Key.
+    </p>
+    <br><br>
+    <h2>Collaborations</h2>
+    <p>
+    Data Commons has benefited greatly from many collaborations. In
+    addition to help from US Department of Commerce (notably the
+    Census Bureau), we have received help from our many academic
+    collaborations, including, UC San Francisco, Stanford University,
+UC Berkeley and Harvard. 
+</p>
+<p>
+We are looking for more collaborators, both for adding new data to
+Data Commons and for building new and interesting applications of Data
+      Commons. Contact us if you are interested in working with us.
+</p>
+
+<h2>Advisory Board</h2>
+
+   We are  fortunate to have the counsel of our Advisory Board,
+which includes:
+<ul>
+  <li> <a href="https://gking.harvard.edu/">Gary King</a>, Director
+   for the Institute for Quantitative Social Science at Harvard
+  University.</li>
+  <li> <a href="https://https://en.wikipedia.org/wiki/Hal_Varian">Hal
+   Varian</a>, Chief Economist, Google. </li>
+   <li><a href="https://energy.stanford.edu/people/arun-majumdar">Arun
+   Majumdar</a>, Director, Precourt Institute for Energy.
+   </ul>
+    
+    
 
     <section>
       <h3>See also</h3>
       <ul>
-        <li><a href="{{ url_for('static.datasets') }}">Ingested datasets</a>
+        <li><a href="{{ url_for('static.datasets') }}">Datasets</a>
```

Comment:
Should we call this Data Sources too? I can also update the URL with a redirect as a follow up.

Reply:
yes, made the change